src/views/download/l10n.json

```diff
@@ -36,6 +36,9 @@
   "download.yesAnswer" : "Yes.",
   "download.onPhone": "Can I install Scratch on my Android phone?",
   "download.onPhoneAnswer" : "No. The current version of Scratch for Android only works on tablets.",
+  "download.howUpdateApp" : "How do I update the Scratch app?",
+  "download.howUpdateAppAnswerApp" : "Open the Google Play store and check for updates. If your installation is managed by school administrators, they will need to update the version and make the update available to managed devices.",
```

Comment:
This is a small note, but maybe the id `howUpdateAppAnswerApp` could be a little more specific, like `download.howUpdateAppAnswerGoogle`?